tests/sentry/test_canonical.py

```diff
@@ -2,13 +2,14 @@
 
 from __future__ import absolute_import, print_function
 
+import unittest
+
 from collections import OrderedDict
 
-from sentry.testutils import TestCase
 from sentry.utils.canonical import CanonicalKeyView, CanonicalKeyDict
 
 
-class CanonicalKeyViewTests(TestCase):
+class CanonicalKeyViewTests(unittest.TestCase):
```

Comment:
If you rename `FooTests` to `TestFoo`, you don't need to inherit from `TestCase` to get the tests discovered. Then using fixtures on each test method should just work

Reply:
I actually only changed these so we could avoid the database activity since they're purely python code tests